src/vsg/state/GraphicsPipeline.cpp

```diff
@@ -398,15 +398,20 @@ void RasterizationState::apply(VkGraphicsPipelineCreateInfo& pipelineInfo) const
 //
 // MultisampleState
 //
-MultisampleState::MultisampleState() :
+MultisampleState::MultisampleState(VkSampleCountFlagBits rasterizationSamples_) :
```

Comment:
The trailing VAR_ looks like typo in the context of the rest of the VSG that doesn't use that convention for function parameter,  I'd rather be explicit with an preceding in_VAR or just a simple renamed like samples.

Also use of two constructors here is a bit overkill when a default value of VkSampleCountFlagBits would have sufficed.  

I will merge this PR and then fix these issues.  I mention here as note for future changes.